<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EXPORT</class>
 <widget class="QWidget" name="EXPORT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EXPORT</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>记录1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>记录2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="le_import">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>2</horstretch>
             <verstretch>2</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>categorylist.json</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_import">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>2</horstretch>
             <verstretch>2</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>导入家具列表</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_export">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>2</horstretch>
             <verstretch>2</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>EXPORT_categorylist.json</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_export">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>2</horstretch>
             <verstretch>2</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>导出家具列表</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
